dlg/dropmake/web/lg_web.py

```diff
@@ -56,6 +55,16 @@ def file_as_string(fname, enc='utf8'):
 MAX_PGT_FN_CNT= 300
 pgt_fn_count = 0
 
+ALGO_PARAMS = [
+    ('min_goal', int),
+    ('ptype', int),
+    ('max_load_imb', int),
+    ('max_cpu', int),
+    ('time_greedy', float),
+    ('deadline', int),
+    ('topk', int),
+    ('swarm_size', int)]
```

Comment:
What about max_mem? I see that in the original list, but I don't really know if it's used by EAGLE or not. It wouldn't harm to keep it in the list, maybe with a small note saying it's only relevant for the old editor